Automattic/woocommerce-payments
# Add support for Czech Republic

Fixes #5971

#### Changes proposed in this Pull Request
Add support for Czech Republic










#### Testing instructions
* On server side checkout 3327-gh-Automattic/woocommerce-payments-server branch
* **Base fees**
* Go to Woo -> Settings -> General and set Store Country to Czech Republic and currency to CZK
* Remove supported countries from the account on server side
* Place an order - base fees should be applied
* **FX fees**
* change store currency to USD
* Remove supported countries from the account on server side
* Place an order - base and FX fees should be applied
* **EPS fees**
* Place an order with EPS
* You will be redirected to a Stripe test page where you can approve/fail the order
* Approve the order - EPS fees should be applied

Continue with all fees defined for Czech Republic .
If the fees does not have 2 decimals, go to `docker/wordpress/wp-content/plugins/woocommerce/i18n/locale-info.php`, look for the currency, and change `num_decimals` to 2 . This will be released with Woo core.




-------------------


**Post merge**
